### ISSUE TITLE
should we keep this?

### ISSUE THREAD

**Issue by EmadAlamoudi** [ICB-DCM/lookahead-study#3]
https://github.com/ICB-DCM/lookahead-study/blob/594a74df3fa69b021c628c32f4f03440316f003b/models/ODE/ODETestFull.py#L30

**Comment by yannikschaelte** [comment]
True, ideally there would be no "absolute paths" anymore in the files, to make things easily reproducible.

**Comment by FelipeR888** [comment]
Valid point, all of my files already define this basepath on one point at the beginning, so if the path changes one only has to change it in that one location. I guess ideally this definition would be taken as an argument, that would remove the absolute paths. In practice is seems a bit tedious to always have to manually pass the path whenever i want to execute the program though.
